ansible/inventory.yml

```diff
@@ -80,6 +80,10 @@ hosts:
 
     - scaleway:
         ubuntu1604-x64-1: {ip: 51.15.76.107}
+    
+    - softlayer:
```

Comment:
weren't these already added in https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/248